app/locales/ua.json

```diff
@@ -0,0 +1,113 @@
+{
+  "Main": {
```

Comment:
this should be `"main": {` (not capital M) 

other keys should be like that as well.